uftrace.c

```diff
@@ -703,6 +705,11 @@ static int parse_option(struct uftrace_opts *opts, int key, char *arg)
 	case 'h':
 		return -3;
 
+	case 'p':
+		opts->pid = strtol(arg, NULL, 0);
+		opts->exename = "";
```

Comment:
Could you explain why this is needed?

Reply:
Maybe we can add this functionality to `live` command and skip recording/replaying.

Reply:
Ok, we don't need to use `record` for this.  I think `live` is a better fit since it'll only talk to a running process.  It can handle this similar to `--list-event`.  Wdyt?

Reply:
I agree, and it also means it can be invoked without explicitly using the `live` command.

As for the syntax, should I keep the `-p` option to trigger client behavior? I was thinking maybe use the target program as the PID value when it is numeric. For example
```
$ uftrace --some-option target # executes and traces the "target" binary
$ uftrace --some-option 1234   # client mode, forwards options to agent in process with PID 1234
```
Would that be relevant?

Reply:
I think it's better to use `-p` explicitly to trigger the client mode.